addons/project_issue/migrations/7.0.1.0/post-migration.py

```diff
@@ -19,7 +19,25 @@
 #
 ##############################################################################
 from openupgrade import openupgrade
-from openerp import pooler
+from openerp import pooler, SUPERUSER_ID
+
+
+def migrate_categories(cr, pool):
+    category_ids = pool['crm.case.categ'].search(
+        cr, SUPERUSER_ID, [('object_id.model', '=', 'project.issue')])
+    for category in pool['crm.case.categ'].browse(
+            cr, SUPERUSER_ID, category_ids):
+        new_category_id = pool['project.category'].create(
+            cr, SUPERUSER_ID, {
+                'name': category.name,
+            })
+        cr.execute('select id from project_issue where categ_id=%s',
```

Comment:
This method is not valid, because you can overwrite other categories that are already changed. Think this:

* old categ_id=1. new categ_id=6. You change all records to categ_id=6.
* old categ_id=6. new categ_id=7. You change all records of old categ_id 1 and 6.